lightcycle-lib/src/main/java/com/soundcloud/lightcycle/util/Preconditions.java

```diff
@@ -0,0 +1,32 @@
+package com.soundcloud.lightcycle.util;
+
+import com.soundcloud.lightcycle.ActivityLightCycle;
+import com.soundcloud.lightcycle.FragmentLightCycle;
+import com.soundcloud.lightcycle.SupportFragmentLightCycle;
+
+/**
+ * Created by yuwono-niko on 11/4/16.
+ */
```

Comment:
Removed! It was generated by Android Studio and didn't realize other files don't have any header.